base/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# The MIT License
+#
+#  Copyright (c) 2017, Oleg Nenashev, Stefan Wallentowitz
```

Comment:
Maybe just LibreCores contributors instead of names. Or FOSSi. I do not care much, but these lists are hard to maintain